js/gui.js

```diff
@@ -6,26 +6,24 @@ var player;
 var level;
 
 
-window.addEventListener("load", function()
-{
-  var container = document.getElementById("tiles");
-  level = createTestLevel();
-  level.showIn(container);
-  
-  var inputOptionsPromise = new Promise(function (resolve) {
-  // input your character here in the form, "src_url": "character_name",
-    resolve({
-         
-      "characters/block.svg": "Block",
-      "characters/sheep.svg": "Intelligent Sheep",
-      "characters/robo.svg": "Robo",
-      "characters/swammy.svg": "Swammy",
-      "characters/monster.svg": "Monster",
-      "characters/elephant.svg": "Elephant",
-      "characters/shadowman.svg": "Shadowman",
-      "characters/santa.svg": "Santa Clause",
```

Comment:
please use the code on the left side  https://github.com/fossasia/labyrinth/pull/223/files -> gui.js and add your character. There should be only one green line on the right. Otherwise all other people need to merge again and handle your change.

Reply:
the same happens to you as I merged #234 If you only change one line, these merges will not need another merge of master in the pull request.